### PR TITLE
Added plan information steps

### LINKED ISSUE #67977: Marketplace: Plan information step hints
The blue-boxed area

Image

Acceptance criteria:

- [ ] Shows the hints (not sure what to call them)

### PR DESCRIPTION
**Pull request by paulopmt1**
#### Proposed Changes

* Added new plan information steps:

	image

- On small screens, it should be displayed with a horizontal scroll (validated with @vinimotaa):

	image

#### Testing Instructions

- After checkout this branch, open the new plugins plan page: http://calypso.localhost:3000/plugins/plans/{site}?flags=plugins-plans-page
- Make sure the current implementation math the design and works as expected

#### Pre-merge Checklist

Related to #67977

### CODE REVIEW

**Review by lupus2k — APPROVED** [review]
LGTM

**Comment by DustyReagan** [comment]
Looks good! Though, are the boxes wider in the Figma? HOg65lHD0QOfaq0QtzKQbC-fi-507%3A10329

File: client/my-sites/plugins/plans/index.tsx [hunk 1]
```diff
@@ -66,6 +68,36 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 					isReskinned
 				/>
 			</div>
+
+			<div className="plugin-plans-step-hints">
```
Comment (lsl) [inline client/my-sites/plugins/plans/index.tsx]: Can you use the [Section component](https://github.com/Automattic/wp-calypso/blob/e99173ff814799c18d52140af9ced68f179ce870/client/my-sites/plugins/education-footer/index.tsx#L47) this?

You can make the headers optional if they aren't currently. Just want to get all the separation margins handled by a single component.

File: client/my-sites/plugins/plans/index.tsx [hunk 2]
```diff
@@ -66,6 +68,36 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
...
+				<section>
```
Comment (lsl) [inline client/my-sites/plugins/plans/index.tsx]: Can you use the [`PromoSection` component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for these?

@vinimotaa Is it ok to ignore the figma around these icons and match the style we used [here](https://wordpress.com/woocommerce-installation/) (what's in red box)?

Screenshot(12)


I recall @SaxonF wanted something like what's in the figma for this but we ended up settling on this layout above for various icons / sizing / mobile handling reasons.

Updating the promo section component to match the new design is also an option but I advise against it.

Comment (paulopmt1) [reply client/my-sites/plugins/plans/index.tsx]: - > Can you use the [PromoSection component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for these?

	Great suggestion @lsl, thanks! [Updated it here](https://github.com/Automattic/wp-calypso/pull/68058/commits/c2656742a1acbe033b96483752b2d5b44fa2b315). 


- > for various icons / sizing / mobile handling reasons

	I recall my call yesterday with @vinimotaa, and one requirement would have less vertical space in both desktop and mobile versions. The [PromoSection component](https://github.com/Automattic/wp-calypso/blob/8f7dc753cfcf1614ad854243f9b2f4e975d22c88/client/my-sites/woocommerce/landing-page.tsx#L217) for default will use more vertical space and won't follow the design at Figma.

	Using the component out of the box, we only need a few lines of custom CSS (which would be great @vinimotaa ):
	
	image

	The downside is the mobile version uses more vertical space @vinimotaa:

	image



- > Updating the promo section component to match the new design is also an option but I advise against it.

	Can you please elaborate more on why? [On Woo implementation](https://github.com/Automattic/wp-calypso/blob/update/67977-new-step-hints/client/my-sites/woocommerce/style.scss#L89-L126) we also have extra CSS code on the promo section. So I'm curious to understand your vision.

File: client/my-sites/plugins/plans/index.tsx (hunk 1 above)
Comment (paulopmt1) [reply client/my-sites/plugins/plans/index.tsx]: Good to know @lsl, [as we replaced the code using the PromoSection](https://github.com/Automattic/wp-calypso/pull/68058/commits/c2656742a1acbe033b96483752b2d5b44fa2b315), I don't think we'll need this anymore.

**Comment by paulopmt1** [comment]
> Looks good! Though, are the boxes wider in the Figma? [HOg65lHD0QOfaq0QtzKQbC-fi-507%3A10329](https://href.li/?https://www.figma.com/file/HOg65lHD0QOfaq0QtzKQbC/?node-id=507%3A10329)

That's right @DustyReagan, as soon it gets merged, I'll update the width to match the previous screen, and it will become bigger. Thanks for noting that!

**Review by DustyReagan — APPROVED** [review]

**Comment by paulopmt1** [comment]
I'll go ahead and merge this task. If we need a follow-up, I'll create another ticket.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7544056

Thank you @paulopmt1 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7544056).

**Pull request merged**